docs/enableexperimental.md

```diff
@@ -0,0 +1,37 @@
+# How to enable experimental features
+
+Fleet supports experimental features that are disabled by default and that can be enabled by the user.
+
+Enabling/disabling experimental features is done using extra environment variables that are available when deploying `rancher/fleet`.
+
+## Enabling an experimental feature
+
+At the moment we're writing this document `Fleet` has OCI storage as an experimental feature.
```

Comment:
```suggestion
At the moment we're writing this document, Fleet has OCI storage as an experimental feature.
```
Why the backticks around `Fleet`? "Fleet" with an uppercase "F" is not the name of a chart nor a repo, it wouldn't appear as such on the command line.